src/current/_includes/common/upgrade-cockroach-version-license-limitations.md

```diff
@@ -0,0 +1,22 @@
+To perform major version upgrades, you must have [a valid license key]({% link {{ page.version.version }}/licensing-faqs.md %}#obtain-a-license).
+
+Patch version upgrades can be performed without a valid license key, with the following limitations:
+
+- The cluster will run without limitations for a 30 day grace period. During that time, alerts are displayed that the cluster needs [a valid license key]({% link {{ page.version.version }}/licensing-faqs.md %}#set-a-license).
```

Comment:
This could become a source of confusion in the future: the 30-day grace period applies only to clusters that existed before the core license deprecation. New clusters created on builds with the updated licensing will have a 7-day grace period. For example, if a cluster is created on version 25.1.0 and later upgraded to 25.1.1, it will still only have the 7-day grace period.